src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -471,6 +474,60 @@ public IActionResult UpdateLogLevel([FromBody] LogRulesRequest request)
             }
         }
 
+        /// <summary>
+        /// Get the enabled log rules.
+        /// </summary>
+        /// <returns>A list of log rules.</returns>
+        [HttpGet]
+        [Route("logrules")]
+        public IActionResult GetLogRules()
+        {
+            // Checks the request is valid.
+            if (!this.ModelState.IsValid)
+            {
+                return ModelStateErrors.BuildErrorResponse(this.ModelState);
+            }
+
+            try
+            {
+                var rules = new List<LogRuleModel>();
+
+                foreach (LoggingRule rule in LogManager.Configuration.LoggingRules)
+                {
+                    string filename = string.Empty;
+
+                    // Retrieve the full path of the current rule's log file.
+                    if (rule.Targets.Any() && rule.Targets.First().GetType().Name == "AsyncTargetWrapper")
```

Comment:
I would say do the `rule.Targets.Any()` check first and if its empty just `continue`.

Reply:
👍